openapi.yaml

```diff
@@ -12192,29 +12151,9 @@ paths:
         explode: false
         schema:
           type: string
-          format: date-time
-        example: to=2024-02-06T00:00:00+01:00
-      - name: filterBy
-        in: query
-        description: |
-          Use this parameter to filter the API's response based on different data dimensions. You can serialize filters in your query to receive more detailed breakdowns of your analytics. You must use `camelCase` for query parameters.
-
-          - If you do not set a value for `filterBy`, the API returns the full dataset for your project.
-          
-          These are the available breakdown dimensions:
-          
-          - `mediaId`: Returns analytics based on the unique identifiers of a video or a live stream.
-          - `mediaType`: Returns analytics based on the type of content. Possible values: `video` and `live-stream`. 
-          - `continent`: Returns analytics based on the viewers' continent. The list of supported continents names are based on the [GeoNames public database](https://www.geonames.org/countries/). Possible values are: `AS`, `AF`, `NA`, `SA`, `AN`, `EU`, `AZ`. 
-          - `country`: Returns analytics based on the viewers' country. The list of supported country names are based on the [GeoNames public database](https://www.geonames.org/countries/).
-          - `deviceType`: Returns analytics based on the type of device used by the viewers. Possible response values are: `computer`, `phone`, `tablet`, `tv`, `console`, `wearable`, `unknown`.
-          - `operatingSystem`: Returns analytics based on the operating system used by the viewers. Response values include `windows`, `mac osx`, `android`, `ios`, `linux`.
-          - `browser`: Returns analytics based on the browser used by the viewers. Response values include `chrome`, `firefox`, `edge`, `opera`.
-        style: form
-        explode: true
-        schema:
-          type: string
-        example: filterBy[continent]=EU&filterBy[country]=FR&filterBy[browser]=Safari&filterBy[browser]=Firefox
+          enum: [hour, day]
+        example: hour
+      - $ref: '#/components/parameters/filterBy'
```

Comment:
I think we are missing the pagination details here:
```
      - $ref: '#/components/parameters/current-page'
      - $ref: '#/components/parameters/page-size'
```

Reply:
Yes, but beware, they're camelCase: `currentPage` and `pageSize`

Reply:
Added ✅ @mioara-api-video `current-page` and `page-size` are just the reference links to the parameter schemas for pagination. The actual params are in `camelCase` ✅